controller_manager/test/test_controller/test_controller.cpp

```diff
@@ -40,6 +40,17 @@ TestController::on_configure(const rclcpp_lifecycle::State & previous_state)
   return rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn::SUCCESS;
 }
 
+rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
+TestController::on_cleanup(
+  const rclcpp_lifecycle::State & previous_state)
+{
+  (void) previous_state;
```

Comment:
can we just command the param instead please?

Reply:
Done